airflow/models/dagrun.py

```diff
@@ -263,10 +294,14 @@ def update_state(self, session=None):
         Determines the overall state of the DagRun based on the state
```

Comment:
Is it not necessary to change the method name? Now does not contain information about tasks. This may not be clear in the future.

Reply:
I don't think so - nothing about it changed, it looks at the TIs associated with the DagRun (based on `self.get_task_instances()`)